sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+| --------------------------- | -------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| `{camera_info}`             | `sensor_msgs::msg::CameraInfo`               | Intrinsic parameters for the calibration cameras . `camera_info` is provided via parameters. |
+| `lidartag/detections_array` | `lidartag_msgs::msg::LidarTagDetectionArray` | LidarTag detections. `lidartag/detections_array` is defined in launcher.                     |
+| `apriltag/detection_array`  | `apriltag_msgs::msg::AprilTagDetectionArray` | AprilTag detections. `apriltag/detection_array` is defined in launcher.                      |
+
+### Output
+
+| Name                   | Type                                             | Description                                          |
+| ---------------------- | ------------------------------------------------ | ---------------------------------------------------- |
+| `filtered_projections` | `visualization_msgs::msg::MarkerArray`           | Publishes the calibration markers for visualization. |
+| `calibration_points`   | `tier4_calibration_msgs::msg::CalibrationPoints` | Publishes the tag points after calibration.          |
+
+### Services
+
+| Name                    | Type                                                  | Description                                                                               |
+| ----------------------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| `extrinsic_calibration` | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocking until the calibration process finishes. |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                          | Type                  | Default Value                    | Description                                                                                                                                               |
+| --------------------------------------------- | --------------------- | -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `calib_rate`                                  | `double`              | `10.0`                           | The rate at which the calibration callback is invoked. This controls the frequency of calibration updates.                                                |
+| `base_frame`                                  | `std::string`         | `base_link`                      | The base_frame is used for visualization.                                                                                                                 |
```

Comment:
`base_frame`

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/50eca415a164ec242cdb629eaacf73654c155162